tools/c7n_azure/setup.py

```diff
@@ -31,5 +31,5 @@
         "custodian.resources": [
             'azure = c7n_azure.entry:initialize_azure']
     },
-    install_requires=["azure", "c7n", "click", "azure-cli-core", "adal~=0.5.0", "futures==3.1.1"],
+    install_requires=["azure", "c7n", "click", "azure-cli-core", "adal~=0.5.0", "pylru", "futures==3.1.1"],
```

Comment:
so we're generally trying to keep custodian under apache license, and this pulls in a gpl dependency, which as a library in python is problematic for the aggregate work. there's a few other lru type implementations out there if your looking for something. python 3 (only) has one builtin to functools https://docs.python.org/3/library/functools.html#functools.lru_cache and there's a few dozen other libraries and recipes for them.